packages/launchpad/src/migration/MajorVersionWelcome.vue

```diff
@@ -9,64 +9,117 @@
         class="bg-white rounded-b max-h-72vh pb-90px overflow-scroll"
       >
         <div class="h-full">
-          <div class="p-16px">
+          <div
+            class="p-16px"
+            data-cy="release-highlights"
+          >
             <h1 class="font-medium mt-4px text-center mb-32px tracking-tighter text-22px text-gray-1000">
               {{ t('majorVersionWelcome.title') }}
             </h1>
             <div class="mb-16px">
               <ExternalLink
-                href="https://on.cypress.io/changelog#11-0-0"
+                href="https://on.cypress.io/changelog#12-0-0"
                 class="font-bold text-indigo-500"
               >
-                11.0.0
+                12.0.0
               </ExternalLink>
               <span class="font-light pl-10px text-gray-500 text-14px">
-                Released {{ versionReleaseDates['11'] }}
+                Released {{ versionReleaseDates['12'] }}
               </span>
             </div>
             <div class="children:mb-16px">
+              <p>
+                For a complete list of updates and breaking changes in v12.0.0, please review our
+                <ExternalLink href="https://on.cypress.io/changelog#12-0-0">
+                  changelog
+                </ExternalLink>.
+              </p>
+
               <h2 class="font-bold text-18px text-jade-1000">
-                Component Testing Released
+                Testing Multi-Origin WorkFlows
               </h2>
 
               <p>
-                Component Testing is now generally available for projects using React, Next.js, Angular, and Vue!
+                Cypress now has full support for testing multiple domains of different origins in a single test with the <ExternalLink href="https://on.cypress.io/origin">
+                  <code>cy.origin()</code>
+                </ExternalLink> command! To take a deep-dive into how this works, read our
+
+                <ExternalLink href="https://www.cypress.io/blog/2022/12/02/the-journey-behind-cy-origin/">
```

Comment:
I'll get that on link created.

Reply:
You may find this naturally but just in case, there's a with a few of these: https://github.com/cypress-io/cypress-services/blob/develop/packages/on/data/links.yml#L598